2020-10-09 01:00:54,808 ERROR 17120 [-/127.0.0.1/-/41ms GET /login?username=2] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password"
name: "ER_PARSE_ERRORError"
pid: 17120
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:15,179 ERROR 15772 [-/127.0.0.1/-/191ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:15,715 ERROR 15772 [-/127.0.0.1/-/40ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:16,532 ERROR 15772 [-/127.0.0.1/-/41ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:17,212 ERROR 15772 [-/127.0.0.1/-/37ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:17,877 ERROR 15772 [-/127.0.0.1/-/44ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:18,489 ERROR 15772 [-/127.0.0.1/-/33ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:25,002 ERROR 15772 [-/127.0.0.1/-/30ms GET /login?username=2&password=1] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:25,879 ERROR 15772 [-/127.0.0.1/-/32ms GET /login?username=2&password=1] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:26,343 ERROR 15772 [-/127.0.0.1/-/30ms GET /login?username=2&password=1] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:26,495 ERROR 15772 [-/127.0.0.1/-/87ms GET /login?username=2&password=1] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:02:26,646 ERROR 15772 [-/127.0.0.1/-/32ms GET /login?username=2&password=1] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '2' AND `password` = '1' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 15772
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:45,353 ERROR 20308 [-/127.0.0.1/-/5ms GET /login?username=2&password=1] nodejs.TypeError: this.app.mysql.find is not a function
    at UserService.login (F:\workercv\app\service\frontend\user.js:5:39)
    at IndexController.login (F:\workercv\app\controller\frontend\index.js:19:50)
    at Object.callFn (F:\workercv\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (F:\workercv\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (F:\workercv\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (F:\workercv\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)
    at next (F:\workercv\node_modules\koa-compose\index.js:45:18)
    at F:\workercv\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)

pid: 20308
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:46,960 ERROR 20308 [-/127.0.0.1/-/0ms GET /login?username=2] nodejs.TypeError: this.app.mysql.find is not a function
    at UserService.login (F:\workercv\app\service\frontend\user.js:5:39)
    at IndexController.login (F:\workercv\app\controller\frontend\index.js:19:50)
    at Object.callFn (F:\workercv\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (F:\workercv\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (F:\workercv\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (F:\workercv\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)
    at next (F:\workercv\node_modules\koa-compose\index.js:45:18)
    at F:\workercv\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)

pid: 20308
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:47,551 ERROR 20308 [-/127.0.0.1/-/1ms GET /login?username=2] nodejs.TypeError: this.app.mysql.find is not a function
    at UserService.login (F:\workercv\app\service\frontend\user.js:5:39)
    at IndexController.login (F:\workercv\app\controller\frontend\index.js:19:50)
    at Object.callFn (F:\workercv\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (F:\workercv\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (F:\workercv\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (F:\workercv\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)
    at next (F:\workercv\node_modules\koa-compose\index.js:45:18)
    at F:\workercv\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)

pid: 20308
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:47,943 ERROR 20308 [-/127.0.0.1/-/0ms GET /login?username=2] nodejs.TypeError: this.app.mysql.find is not a function
    at UserService.login (F:\workercv\app\service\frontend\user.js:5:39)
    at IndexController.login (F:\workercv\app\controller\frontend\index.js:19:50)
    at Object.callFn (F:\workercv\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (F:\workercv\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (F:\workercv\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (F:\workercv\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)
    at next (F:\workercv\node_modules\koa-compose\index.js:45:18)
    at F:\workercv\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)

pid: 20308
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:48,057 ERROR 20308 [-/127.0.0.1/-/0ms GET /login?username=2] nodejs.TypeError: this.app.mysql.find is not a function
    at UserService.login (F:\workercv\app\service\frontend\user.js:5:39)
    at IndexController.login (F:\workercv\app\controller\frontend\index.js:19:50)
    at Object.callFn (F:\workercv\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (F:\workercv\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (F:\workercv\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (F:\workercv\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)
    at next (F:\workercv\node_modules\koa-compose\index.js:45:18)
    at F:\workercv\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)

pid: 20308
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:48,347 ERROR 20308 [-/127.0.0.1/-/1ms GET /login?username=2] nodejs.TypeError: this.app.mysql.find is not a function
    at UserService.login (F:\workercv\app\service\frontend\user.js:5:39)
    at IndexController.login (F:\workercv\app\controller\frontend\index.js:19:50)
    at Object.callFn (F:\workercv\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (F:\workercv\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (F:\workercv\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (F:\workercv\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)
    at next (F:\workercv\node_modules\koa-compose\index.js:45:18)
    at F:\workercv\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)

pid: 20308
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:53,180 ERROR 20308 [-/127.0.0.1/-/0ms GET /login?username=2] nodejs.TypeError: this.app.mysql.find is not a function
    at UserService.login (F:\workercv\app\service\frontend\user.js:5:39)
    at IndexController.login (F:\workercv\app\controller\frontend\index.js:19:50)
    at Object.callFn (F:\workercv\node_modules\egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (F:\workercv\node_modules\egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (F:\workercv\node_modules\@eggjs\router\lib\utils.js:12:21)
    at wrappedController (F:\workercv\node_modules\@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)
    at next (F:\workercv\node_modules\koa-compose\index.js:45:18)
    at F:\workercv\node_modules\@eggjs\router\lib\router.js:190:18
    at dispatch (F:\workercv\node_modules\koa-compose\index.js:44:32)

pid: 20308
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:54,796 ERROR 12552 [-/127.0.0.1/-/44ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '0' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '0' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12552
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:55,947 ERROR 12552 [-/127.0.0.1/-/197ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '0' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '0' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12552
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:56,393 ERROR 12552 [-/127.0.0.1/-/332ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '0' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '0' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1"
headerSent: true
name: "ER_BAD_FIELD_ERRORError"
pid: 12552
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:56,695 ERROR 12552 [-/undefined/-/405ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '0' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '0' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1"
headerSent: true
name: "ER_BAD_FIELD_ERRORError"
pid: 12552
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:56,701 ERROR 12552 [-/undefined/-/269ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '0' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '0' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1"
headerSent: true
name: "ER_BAD_FIELD_ERRORError"
pid: 12552
hostname: DESKTOP-EF7P7GO

2020-10-09 01:03:56,731 ERROR 12552 [-/127.0.0.1/-/90ms GET /login?username=2] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '0' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '0' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `0` = '2' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12552
hostname: DESKTOP-EF7P7GO

2020-10-09 01:04:33,032 ERROR 5596 [-/127.0.0.1/-/36ms GET /login?username=0] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 5596
hostname: DESKTOP-EF7P7GO

2020-10-09 01:04:34,070 ERROR 5596 [-/127.0.0.1/-/27ms GET /login?username=0] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 5596
hostname: DESKTOP-EF7P7GO

2020-10-09 01:05:13,731 ERROR 5596 [-/127.0.0.1/-/29ms GET /login?username=13184845054&password=123] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 5596
hostname: DESKTOP-EF7P7GO

2020-10-09 01:05:23,934 ERROR 5596 [-/127.0.0.1/-/29ms GET /login?username=13184845054&password=123] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 5596
hostname: DESKTOP-EF7P7GO

2020-10-09 01:05:26,464 ERROR 13084 [-/127.0.0.1/-/98ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ?"
name: "ER_PARSE_ERRORError"
pid: 13084
hostname: DESKTOP-EF7P7GO

2020-10-09 01:05:27,066 ERROR 13084 [-/127.0.0.1/-/50ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ?"
name: "ER_PARSE_ERRORError"
pid: 13084
hostname: DESKTOP-EF7P7GO

2020-10-09 01:05:27,756 ERROR 13084 [-/127.0.0.1/-/45ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ?"
name: "ER_PARSE_ERRORError"
pid: 13084
hostname: DESKTOP-EF7P7GO

2020-10-09 01:05:28,765 ERROR 13084 [-/127.0.0.1/-/55ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ?"
name: "ER_PARSE_ERRORError"
pid: 13084
hostname: DESKTOP-EF7P7GO

2020-10-09 01:05:40,053 ERROR 13084 [-/127.0.0.1/-/48ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ?"
name: "ER_PARSE_ERRORError"
pid: 13084
hostname: DESKTOP-EF7P7GO

2020-10-09 01:06:27,615 ERROR 1400 [-/127.0.0.1/-/60ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 1400
hostname: DESKTOP-EF7P7GO

2020-10-09 01:06:28,669 ERROR 1400 [-/127.0.0.1/-/50ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 1400
hostname: DESKTOP-EF7P7GO

2020-10-09 01:06:29,240 ERROR 1400 [-/127.0.0.1/-/55ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 1400
hostname: DESKTOP-EF7P7GO

2020-10-09 01:06:30,010 ERROR 1400 [-/127.0.0.1/-/409ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
headerSent: true
name: "ER_PARSE_ERRORError"
pid: 1400
hostname: DESKTOP-EF7P7GO

2020-10-09 01:06:30,353 ERROR 1400 [-/undefined/-/624ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
headerSent: true
name: "ER_PARSE_ERRORError"
pid: 1400
hostname: DESKTOP-EF7P7GO

2020-10-09 01:06:30,368 ERROR 1400 [-/127.0.0.1/-/381ms GET /login?username=13184845054] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: select * from user where phone = ? and password = ?
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '? and password = ?' at line 1"
sqlState: "42000"
index: 0
sql: "select * from user where phone = ? and password = ?"
name: "ER_PARSE_ERRORError"
pid: 1400
hostname: DESKTOP-EF7P7GO

2020-10-09 01:07:24,506 ERROR 17356 [-/127.0.0.1/-/172ms GET /login?username=13184845054] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 17356
hostname: DESKTOP-EF7P7GO

2020-10-09 01:07:25,357 ERROR 17356 [-/127.0.0.1/-/40ms GET /login?username=13184845054] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` IS NULL LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` IS NULL LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 17356
hostname: DESKTOP-EF7P7GO

2020-10-09 01:07:33,407 ERROR 17356 [-/127.0.0.1/-/40ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 17356
hostname: DESKTOP-EF7P7GO

2020-10-09 01:07:34,975 ERROR 17356 [-/127.0.0.1/-/43ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 17356
hostname: DESKTOP-EF7P7GO

2020-10-09 01:08:21,339 ERROR 17356 [-/127.0.0.1/-/222ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 17356
hostname: DESKTOP-EF7P7GO

2020-10-09 01:09:46,079 ERROR 2364 [-/127.0.0.1/-/78ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 2364
hostname: DESKTOP-EF7P7GO

2020-10-09 01:09:47,042 ERROR 2364 [-/127.0.0.1/-/29ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 2364
hostname: DESKTOP-EF7P7GO

2020-10-09 01:09:48,483 ERROR 2364 [-/127.0.0.1/-/26ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 2364
hostname: DESKTOP-EF7P7GO

2020-10-09 01:10:21,881 ERROR 12056 [-/127.0.0.1/-/269ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12056
hostname: DESKTOP-EF7P7GO

2020-10-09 01:10:22,831 ERROR 12056 [-/127.0.0.1/-/215ms GET /login?username=13184845054&password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` = '13184845054' AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12056
hostname: DESKTOP-EF7P7GO

2020-10-09 01:10:27,870 ERROR 12056 [-/127.0.0.1/-/30ms GET /login?password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12056
hostname: DESKTOP-EF7P7GO

2020-10-09 01:10:29,644 ERROR 12056 [-/127.0.0.1/-/31ms GET /login?password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12056
hostname: DESKTOP-EF7P7GO

2020-10-09 01:10:30,960 ERROR 12056 [-/127.0.0.1/-/36ms GET /login?password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12056
hostname: DESKTOP-EF7P7GO

2020-10-09 01:10:37,452 ERROR 12056 [-/127.0.0.1/-/42ms GET /login?password=123] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'username' in 'where clause'
    at Query.Sequence._packetToError (F:\workercv\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (F:\workercv\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (F:\workercv\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (F:\workercv\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (F:\workercv\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (F:\workercv\node_modules\mysql\lib\Pool.js:199:23)
    at F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (F:\workercv\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (F:\workercv\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (F:\workercv\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (F:\workercv\node_modules\co\index.js:65:19)
    at F:\workercv\node_modules\co\index.js:54:5
    sql: SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'username' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT * FROM `user` WHERE `username` IS NULL AND `password` = '123' LIMIT 0, 1"
name: "ER_BAD_FIELD_ERRORError"
pid: 12056
hostname: DESKTOP-EF7P7GO

